### ISSUE TITLE
Dependency installation failed!

### ISSUE THREAD

**Issue by hieutrandn9889** [FluxionNetwork/fluxion#578]
I clone github to my computer in kali linux 2018.2 
Run fluxion and get this issue as below:
./fluxion.sh: line 248: format_center_literals: command not found

./fluxion.sh: line 255: format_center_literals: command not found

./fluxion.sh: line 263: installer_utils_check_update: command not found

./fluxion.sh: line 282: installer_utils_check_dependencies: command not found
./fluxion.sh: line 283: installer_utils_run_dependencies: command not found

Dependency installation failed!
Press enter to retry, ctrl+c to exit...

**Comment by rad4day** [comment]
Couldn't reproduce.
fresh kali 2018.2 lite install
updated + upgraded + installed git

**Comment by hieutrandn9889** [comment]
Hello @rad4day ,

I have just updated, upgrade + installed git. But it still display and doesn't work.
How to view log when starting this tool ? I want to show you the log when running it

**Comment by deltaxflux** [comment]
Are you able to install the dependencies manully? Like:
`sudo apt-get install hostapd`

**Comment by Tortuga619** [comment]
@deltaxflux so I was having alot of trouble with downloading anything on that distro so I tried it on my other computer im running ubuntu 18.04 bionic beaver on oracle virtualbox on my windows 7 64bit. So I downloaded fluxion 5 rev6 and manually upgraded to the lates aircrack-ng, downloaded cowpatty,rhcp services ,and mdk3 and then downloaded all dependencies through ./fluxion.sh -i  I am using TL-WN722N v1 adapter with atheros chipset so I get the 4way handshake when I use aircrack-ng option and when I go to the captive portal attack I start the attack and I'm missing the dhcp and dns service windows they dont come up at all and I show 0 client is tried the attack on my own service and computer and i get the fake access point with the same name but I connect to it and it doesnt rederct me to the fluxion password window. Do I have to configure the dhcp? Or what can be the problem?
